src_c/draw.c

```diff
@@ -108,6 +226,7 @@ aaline(PyObject *self, PyObject *arg, PyObject *kwargs)
     int drawn_area[4] = {INT_MAX, INT_MAX, INT_MIN,
                          INT_MIN}; /* Used to store bounding box values */
     Uint32 color;
+    int (*set_func)(SDL_Surface*, int, int, Uint32);
```

Comment:
Looks like we need to set a default value for `set_func` here to please gcc. I guess something like:

```suggestion
    int (*set_func)(SDL_Surface*, int, int, Uint32) = &set_at_32bit_clip;;
```